### DISCUSSION TITLE
蜿蜒针迹能否这样?

### DISCUSSION THREAD

**Discussion by succeed909** [inkstitch/inkstitch#3296] [Ideas]
蜿蜒针迹能否如图这样的走法？边缘整齐，每个针迹点的周围相隔距离基本一致。
如图，即每个针迹点间隔几乎一样，图2为图1的针迹点。
![1](https://github.com/user-attachments/assets/96482575-6ad0-45e7-9d8c-449143cee3db)
![2](https://github.com/user-attachments/assets/5bbb7624-53e7-4333-8f8b-724c12718bf2)

**Comment by succeed909** [comment]
这个提议有结果了吗？

**Comment by bkmgit** [comment]
See https://inkstitch.org/docs/stitches/meander-fill/

例如，尝试翻译 https://translate.baidu.com
